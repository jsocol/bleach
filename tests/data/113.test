<div style=content:url(data:image/svg+xml,%3Csvg/%3E);visibility:hidden onload=alert(41)></div>//["'`-->]]>]</div>
