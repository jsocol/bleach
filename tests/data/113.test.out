<div style="content:url(data:image/svg+xml,%3Csvg/%3E);visibility:hidden"></div>//["'`--&gt;]]&gt;]
